build-tools/automation/azure-pipelines.yaml

```diff
@@ -1126,3 +1126,28 @@ stages:
   - template: yaml-templates\run-timezoneinfo-tests.yaml
     parameters:
       node_id: 3
+
+  # Check - "Xamarin.Android (Finalize Installers Queue Vsix Signing)"
+  - job: queue_vsix_signing
```

Comment:
We've moved things around a bit in master, but the Jenkins sign job does exist on d16-8:
https://github.com/xamarin/xamarin-android/blob/d16-8/build-tools/automation/azure-pipelines.yaml#L435

We should replace that job with the changes below.

Reply:
I made the appropriate fixups per https://github.com/xamarin/xamarin-android/pull/5392/commits/cbca46c14eea0ed6ae5814e14db6fd8848046f6f